lisa/tools/lsvmbus.py

```diff
@@ -45,7 +46,13 @@ def __repr__(self) -> str:
         return self.__str__()
 
 
-class VmBusDevice:
+class VmBusDeviceResult:
```

Comment:
Why need the new result class?

Reply:
The `VmBusDevice` class is very tied to Linux. So I created a parent class with the basic variables we need to represent a Lsvmbus device

Reply:
Name the VmBusDevice as LinuxVmBusDeviceParser, and not inherit from VmBusDevice. So it would be clear, the VmBusDevice is the data structure, and the Parser is the logic.